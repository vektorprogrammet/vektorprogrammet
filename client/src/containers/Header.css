.logo{
    display: block;
    margin: 0 auto;
    max-height: 100%;
    max-width: 100%;
    height: 85px;
}

.main-navigation {
    margin: 4px 0 8px;
}

.main-navigation a {
    font-size: 18px;
    color: #4A5155;
    margin-right: 30px;
}

.main-navigation a:active {
    font-weight: 500;
}

.main-navigation a:active, .main-navigation a:hover {
  color: #0071CD;
}
