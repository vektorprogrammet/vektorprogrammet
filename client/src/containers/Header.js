import React, {Component} from 'react';
import { Link } from 'react-router-dom';
import { Grid, Image } from 'semantic-ui-react';
import logo from '../images/logo_condensed.png';
import './Header.css';

class Header extends Component {
  render() {
    return (
    <Grid>
      <Grid.Row centered>
        <Link to={'/'}>
          <Image className="logo" src={logo} />
        </Link>
      </Grid.Row>
      <Grid.Row centered>
        <nav className="main-navigation">
          <Link to={'/assistenter'}>Assistenter</Link>
          <Link to={'/team'}>Team</Link>
          <Link to={'/laerere'}>LÃ¦rere</Link>
          <Link to={'/foreldre'}>Foreldre</Link>
          <Link to={'/om-oss'}>Om oss</Link>
        </nav>
      </Grid.Row>
    </Grid>
    );
  }
}

export default Header;
