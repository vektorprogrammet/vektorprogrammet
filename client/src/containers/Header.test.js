import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'
import Header from './Header';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render((
      <BrowserRouter>
        <Header />
      </BrowserRouter>
  ), div);
});

it('renders correctly', () => {
  const tree = renderer.create((
      <BrowserRouter>
        <Header />
      </BrowserRouter>
  )).toJSON();
  expect(tree).toMatchSnapshot();
});
