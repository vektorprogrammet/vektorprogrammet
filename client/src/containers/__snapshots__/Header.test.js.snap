// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui grid"
>
  <div
    className="centered row"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt={undefined}
        className="ui image logo"
        height={undefined}
        src="logo_condensed.png"
        width={undefined}
      />
    </a>
  </div>
  <div
    className="centered row"
  >
    <nav
      className="main-navigation"
    >
      <a
        href="/assistenter"
        onClick={[Function]}
      >
        Assistenter
      </a>
      <a
        href="/team"
        onClick={[Function]}
      >
        Team
      </a>
      <a
        href="/laerere"
        onClick={[Function]}
      >
        LÃ¦rere
      </a>
      <a
        href="/foreldre"
        onClick={[Function]}
      >
        Foreldre
      </a>
      <a
        href="/om-oss"
        onClick={[Function]}
      >
        Om oss
      </a>
    </nav>
  </div>
</div>
`;
